src/modules/rc_update/rc_update.cpp

```diff
@@ -155,6 +154,12 @@ void RCUpdate::parameters_updated()
 
 	update_rc_functions();
 
+	_rc_calibrated = _param_rc_chan_cnt.get() > 0
+			 && (_param_rc_map_throttle.get() > 0
```

Comment:
What about a rover?

What it be enough to start with just RC_CHAN_CNT > 0 and RC_MAP_THROTTLE?

Reply:
I thought about that case. Just one of the 4 main functions needs to be mapped:
channel count > 0 and (throttle or roll or pitch or yaw). Did I do a mistake?